btr-web/btr-main-app/pages/requestToOmit.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import { useOmitIndividual } from '@/stores/omit-individual'
+
+const omitIndividual = useOmitIndividual()
+const cpRef = ref()
+
+async function handleDoneButtonClick () {
+  // TODO: #22111 Add logic when API is added
+  await cpRef.value.validate()
+  omitIndividual.errors = [...cpRef.value.completingPartyForm.errors]
+  if (omitIndividual.errors.length === 0) {
+    // eslint-disable-next-line no-console
+    console.log('ok to submit')
+  } else {
+    console.error('Fix errors before submitting, ', cpRef.value.completingPartyForm.errors)
+  }
+}
+</script>
+
+<template>
+  <div>
+    <div data-cy="request-to-omit-header">
+      <h3 class="text-xl font-bold mb-2.5">
+        {{ $t('general.regName') }}
+      </h3>
+      <h1 class="text-2xl font-bold mb-4" data-cy="request-to-omit-title">
+        {{ $t('labels.requestOmit.header') }}
+      </h1>
+      <p class="text-base mb-4" data-cy="request-to-omit-text">
+        {{ $t('texts.requestOmitText') }}
+      </p>
+      <BcrosHelpTip
+        :title="$t('helpTitles.requestOmit.closed')"
+        :title-expanded="$t('helpTitles.requestOmit.expanded')"
+      >
+        <template #default>
+          <BcrosI18HelperLink translation-path="helpTexts.requestOmit.detail" />
+        </template>
+      </BcrosHelpTip>
+    </div>
+
+    <BcrosSection
+      data-cy="completing-party"
+      :section-title="$t('labels.requestOmit.completingParty')"
+      section-title-icon="i-mdi-account-supervisor-circle-outline"
+      rounded-bot
+      rounded-top
+    >
+      <BcrosCompletingParty ref="cpRef" v-model="omitIndividual.completingParty" />
+    </BcrosSection>
+
+    <div class="mt-5 w-full mb-5 h-[30px]">
```

Comment:
Need to remove the 'h-[30px]' here otherwise the button gets pushed outside the div and off to the side since its height is bigger

![Screenshot 2024-09-16 at 9 16 01 AM](https://github.com/user-attachments/assets/e08b572e-c4c9-49e1-a3a1-8da68d27a2b1)

Reply:
I've made a change to fix this, however it's not removing the h. I changed the height so that it properly contains the button but whether the button floats or is positioned absolute to the container it causes issues with the footer without the height on the containing div (float cuts the footer smaller, absolute overlaps the footer). What you're seeing though is that the footer div is full-w without the max-w-1360px that is enforced in the page itself so I've added that